bindings/fortran/dlite.f90

```diff
@@ -120,11 +120,11 @@ end function dlite_storage_is_writable_c
   ! --------------------------------------------------------
   ! C interface for DLiteInstance
   ! --------------------------------------------------------
-  type(c_ptr) function dlite_instance_create_from_id_c(metaid, dims, id) &
+  type(c_ptr) function dlite_instance_create_from_id_c(metaid, shape, id) &
```

Comment:
Keep `dims` here, since it refer to the dimensions of the instance.

Reply:
done